Human/README.md

```diff
@@ -24,11 +24,11 @@ Given a compendium of gene expression partitions, how will increasing the number
 **Conclusions:**
 Having some number of technical variation confounded our ability to extract our original biological signal and we need to apply correction.  However, as the number of technical variation increased, it became easier to discover the underlying biological patterns and applying correction removes some of the biology of interest.  Our ability to extract biological signal is defined by our ability to retain the structure of the biological data -- in other words, is our representation of the simulated data with a single partition similar to the representation with multiple partitions?  
 
-![Similarity_Human_sample](https://github.com/greenelab/simulate-expression-compendia/blob/master/results/Human_sample_lvl_sim_svcca.png)
+![Similarity_Human_sample](https://github.com/greenelab/simulate-expression-compendia/blob/master/Human/results/Human_sample_lvl_sim_svcca_limma.png)
 
 
 ## Experiment-level simulation experiment 
-Scripts can be found in [scripts/Human_experiment_lvl_sim](https://github.com/greenelab/simulate-expression-compendia/tree/master/Human_analysis/Human_experiment_lvl_sim)  
+Scripts to run this analysis can be found in [Human_experiment_lvl_sim.ipynb](https://github.com/greenelab/simulate-expression-compendia/blob/master/Human/Human_experiment_lvl_sim.ipynb)  
```

Comment:
This is great! Having everything live in a single script is better than pointing to an entire directory, because then the order of scripts is abundantly clear.